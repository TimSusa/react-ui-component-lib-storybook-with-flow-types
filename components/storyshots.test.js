/*
  Run the storyshots feature when running tests
  Ref: https://github.com/storybooks/storyshots
*/
import initStoryshots from '@storybook/addon-storyshots';
initStoryshots();
