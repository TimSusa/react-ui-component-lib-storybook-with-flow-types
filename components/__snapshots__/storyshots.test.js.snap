exports[`Storyshots Button with a label and an icon 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }>
  Settings ⚙️
</button>
`;

exports[`Storyshots Button with label 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }>
  Hello Button
</button>
`;

exports[`Storyshots Button with some emoji 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }>
  😀 😎 👍 💯
</button>
`;

exports[`Storyshots H1 with label 1`] = `
<h1
  style={
    Object {
      "fontSize": "2em",
      "marginBottom": "0.25em",
    }
  }>
  Header text
</h1>
`;

exports[`Storyshots UserDetails default 1`] = `
<div>
  <p>
    Id: 
    1
  </p>
  <p>
    Name: 
    Joe
  </p>
  <p>
    Num: 
    42
  </p>
  <p>
    Option: 
    foo
  </p>
</div>
`;
